shop-data/src/main/AndroidManifest.xml

```diff
@@ -1,5 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.ui.shop_data">
+    <uses-permission android:name="android.permission.INTERNET"/>
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE"/>
```

Comment:
As I understand, currently we don't need `.ACCES_NETWORK_STATE`. Please remove it

Reply:
fixed